MASH-FRET/source/mod-trace-processing/management/traceManager.m

```diff
@@ -397,10 +398,21 @@ function openMngrTool(h_fig)
         'Position', [xNext yNext w_pop h_but], ...
         'TooltipString', 'select a molecule tag', ...
         'FontUnits', 'pixels', ...
-        'FontSize', fntS);
+        'FontSize', fntS, 'Callback', {@popup_molTag_Callback,h_fig});
+    
+    % added by MH, 24.4.2019
+    xNext = xNext + w_pop + mg;
+    hexclr = h.tm.molTagClr{get(h.tm.popup_molTag,'value')}(2:end);
+    
+    h.tm.edit_tagClr = uicontrol('Style','edit','Parent', ...
```

Comment:
Can we add a small pushbutton next to the edit field of the HEX color that opens the built-in `uisetcolor` function to let the user choose a color manually (without knowing the HEX string a priori)? The edit field would then display the chosen color.

Reply:
<3 <3 !!
I also got this idea but thought it was "too much", will be aded

Reply:
Thank you very much, I did not know `uisetcolor`. The feature is now on in Trace manager and in Project options

Reply:
Great, thanks! 👍